scripts/convert_backtestdata.py

```diff
@@ -0,0 +1,183 @@
+import gzip
+
+from freqtrade.arguments import Arguments
+from freqtrade import misc
+from freqtrade.logger import Logger
+from pandas import DataFrame
+
+import dateutil.parser
+
+logger = Logger(name="Convert data", level=10).get_logger()
+
+
+def load_old_file(filename) -> (List[Dict], bool):
+    if not path.isfile(filename):
+        logger.warning("filename %s does not exist", filename)
+        return (None, False)
+    logger.debug('Loading ticker data from file %s', filename)
+
+    pairdata = None
+
+    if filename.endswith('.gz'):
+        logger.debug('Loading ticker data from file %s', filename)
+        is_zip = True
+        with gzip.open(filename) as tickerdata:
+            pairdata = json.load(tickerdata)
+    else:
+        is_zip = False
+        with open(filename) as tickerdata:
+            pairdata = json.load(tickerdata)
+    return (pairdata, is_zip)
+
+
+def parse_old_backtest_data(ticker) -> DataFrame:
+    """
+    Reads old backtest data
+    Format: "O": 8.794e-05,
+            "H": 8.948e-05,
+            "L": 8.794e-05,
+            "C": 8.88e-05,
+            "V": 991.09056638,
+            "T": "2017-11-26T08:50:00",
+            "BV": 0.0877869
+    """
+
+    columns = {'C': 'close', 'V': 'volume', 'O': 'open',
+               'H': 'high', 'L': 'low', 'T': 'date'}
+
+    frame = DataFrame(ticker) \
+        .rename(columns=columns)
+    if 'BV' in frame:
+        frame.drop('BV', 1, inplace=True)
+    if not 'date' in frame:
+        logger.warning("Date not in frame - probably not a Ticker file")
+        return None
+    frame.sort_values('date', inplace=True)
+    return frame
+
+
+def convert_dataframe(frame: DataFrame):
+    """Convert dataframe to new format"""
+    # reorder columns:
+    cols = ['date', 'open', 'high', 'low', 'close', 'volume']
+    frame = frame[cols]
+
+    frame['date'] = frame['date'].apply(
+        lambda d: int(dateutil.parser.parse(d).timestamp()) * 1000)
+    frame['date'] = frame['date'].astype(int)
+    # Convert columns one by one to preserve type.
+    by_column = [frame[x].values.tolist() for x in frame.columns]
+    return list(list(x) for x in zip(*by_column))
+
+
+def convert_file(filename: str, filename_new: str):
+    """Converts a file from old format to ccxt format"""
+    (pairdata, is_zip) = load_old_file(filename)
+    if pairdata and type(pairdata) is list and len(pairdata) > 0:
+        if type(pairdata[0]) is list:
+            logger.error("pairdata for %s already in new format", filename)
+            return 1
+
+    frame = parse_old_backtest_data(pairdata)
+    # Convert frame to new format
+    if frame is not None:
+        frame1 = convert_dataframe(frame)
+        misc.file_dump_json(filename_new, frame1, is_zip)
+
+
+def convert_main(args: Namespace) -> None:
+    """
+    converts a folder given in --datadir from old to new format to support ccxt
+    """
+
+    workdir = args.datadir if args.datadir.endswith("/") else args.datadir + "/"
```

Comment:
Don't rely on `/`. This will break on windows. Use `os.path` for pathname manipulations